cli/graphql.md

```diff
@@ -2358,6 +2439,118 @@ mutation Delete($noteId: ID!) {
 }
 ```
 
+## Automatically Import Existing DataSources
+
+The Amplify CLI currently supports importing serverless Amazon Aurora MySQL 5.6 databases. Follow these instructions to get started:
```

Comment:
"Follow these instructions to get started:" --> The following instruction show how to create a Amazon Aurora Serverless database, import this database as a GraphQL data source and test it.

Reply:
fixed